addons/web/static/src/views/kanban/kanban_cover_image_dialog.xml

```diff
@@ -18,21 +18,21 @@
                 </t>
             </div>
             <t t-else="">There is no available image to be set as cover.</t>
-            <FileInput t-if="state.selectFile"
-                acceptedFileExtensions="'image/*'"
-                autoOpen="true"
-                hidden="true"
-                onUpload.bind="onUpload"
-                resModel="props.record.resModel"
-                resId="props.record.resId"
-            />
             <t t-set-slot="footer" t-slot-scope="dialog">
                 <button t-if="attachments.length" class="btn btn-primary" t-on-click="setCover" t-att-disabled="!state.selectedAttachmentId">
                     Select
                 </button>
-                <button class="btn btn-primary" t-on-click="uploadImage">
-                    Upload and Set
-                </button>
+                <FileInput
+                acceptedFileExtensions="'image/*'"
+                autoOpen="false"
+                hidden="false"
+                onUpload.bind="onUpload"
+                resModel="props.record.resModel"
+                resId="props.record.resId">
+                    <button class="btn btn-primary" t-on-click="uploadImage">
+                        Upload and Set
+                    </button>
+                </FileInput>
```

Comment:
Indentation :nerd_face: 
Usually I like having one attribute next to the tag and the rest indented one level more than the tag. Like it was before your changes

Reply:
@pkyriakou, the changes requested are done.